Source/AhriAndBear/Private/Component/AABSurvivalComponent.cpp

```diff
@@ -47,3 +47,8 @@ FORCEINLINE void UAABSurvivalComponent::TickStat(float& currentValue, int maxVal
 	currentValue = FMath::Clamp<float>(currentValue + delta, 0, maxValue);
 }
 
+void UAABSurvivalComponent::AddSurvivalData(const FSurvivalData& value)
+{
+		
+}
+
```

Comment:
@greenstack I added this function to fulfill the EventTrigger's need, please implement it or rename it to the function you created, thx

Reply:
I'll take care of this in the `jn_survival_ui` branch once this pull request is merged due to API changes in that branch.